tests/scripts/kubeadm.sh

```diff
@@ -130,8 +131,10 @@ case "${1:-}" in
             install_master
         ;;
         node)
-            if [ "$#" -eq 5 ]; then
-                install_node $3 $4 $5
+            if [[ $KUBE_VERSION == v1.8* ]] && [ "$#" -eq 7 ]; then
```

Comment:
should this if statement now just check for 5 or 7 arguments? Then you don't need the `elif` clause that calls the same thing

Reply:
yes, good idea, that way we dont have to update the script when k8s 1.9 is out.